homeassistant/components/lamarzocco/lm_client.py

```diff
@@ -0,0 +1,136 @@
+"""La Marzocco Cloud API client."""
+from collections.abc import Callable, Mapping
+import logging
+from typing import Any
+
+from lmcloud import LMCloud
+from lmcloud.exceptions import BluetoothConnectionFailed
+
+from homeassistant.components import bluetooth
+from homeassistant.const import CONF_HOST, CONF_MAC, CONF_NAME, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+
+from .const import CONF_MACHINE
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class LaMarzoccoClient(LMCloud):
+    """Keep data for La Marzocco entities."""
+
+    _bt_disconnected = False
+
+    def __init__(
+        self,
+        hass: HomeAssistant,
+        entry_data: Mapping[str, Any],
+        callback: Callable[[], None] | None = None,
+    ) -> None:
+        """Initialise the LaMarzocco entity data."""
+        super().__init__()
+        self._entry_data = entry_data
+        self.hass = hass
+        self._callback_websocket_notify = callback
```

Comment:
Where is this variable used? If it is used only in the parent, you should pass it on the `super.init` call